docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
       - ./data:/srv/${WP_ENV}/jahia2wp/data
       - ./requirements:/srv/${WP_ENV}/jahia2wp/requirements
       - ./src:/srv/${WP_ENV}/jahia2wp/src
-      - ./Makefile:/srv/${WP_ENV}/jahia2wp/Makefile
+      - ./Makefile.mgmt:/srv/${WP_ENV}/jahia2wp/Makefile
```

Comment:
clever trick. Like it very much !!!

And indeed, it makes the two Makefiles more legible